lua/decisive.lua

```diff
@@ -1,3 +1,12 @@
+local function check_version()
+  if vim.version().major < 0 or vim.version().minor < 10 then
```

Comment:
could you change that to "if major == 0 and minor < 10"?

Reply:
Fixed